modules/apps/headless-apio/forms/forms-apio-impl/src/main/java/com/liferay/forms/apio/internal/architect/resource/FormInstanceNestedCollectionResource.java

```diff
@@ -70,8 +81,25 @@ public String getName() {
 	public ItemRoutes<DDMFormInstance, Long> itemRoutes(
 		ItemRoutes.Builder<DDMFormInstance, Long> builder) {
 
+		GetRoute fetchLatestDraftRoute = new FetchLatestDraftRoute();
+
+		BiFunction<Credentials, Long, Boolean> hasPermissionFunction =
+			(credentials, identifier) -> Try.fromFallible(
+				() -> _hasPermission.forAddingIn(
+					FormInstanceRecordIdentifier.class
+				).apply(
+					credentials, identifier
+				)
+			).orElse(
+				false
+			);
+
 		return builder.addGetter(
 			_ddmFormInstanceService::getFormInstance
+		).addCustomRoute(
+			fetchLatestDraftRoute, this::_fetchLatestDraft, CurrentUser.class,
+			FormInstanceRecordIdentifier.class, hasPermissionFunction,
+			FetchLatestDraftForm::buildForm
```

Comment:
@nhpatt this form is not needed since it's a Get operation, I only created it to avoid errors. Is there a better way to do it?

Reply:
No, at the moment. I created some overloaded methods without the form parameter but it was too much code that is going to change in the future (old API will continue to work)

Reply:
Ok, no problem.